extensions/adobe/experience/target/classification.schema.json

```diff
@@ -0,0 +1,96 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/target/classification",
```

Comment:
@surbhi114 , Are you defining a class here or just a data type ?

Reply:
I would like it to be a class that can be used for creating schemas, and later on we can query the schema registry using the included class to get back the schema.

Reply:
Ok, In that case, I would suggest you to fully qualify the property namespaces. We only use xdm: for a property within an internal Adobe solution XDM object if that object is a data type. 

Here's an example of a fully qualified property namespace. Observe the properties in this link.

https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

Please work within your team to come up with the right hierarchy for your use cases. 

Also, as a general rule, we try to keep only those properties in a class which are mandatory to define the entity the class represents. Rest of the properties should sit under respective field groups as per their logical groupings.